rustworkx/py.typed

```diff
@@ -0,0 +1 @@
+partial
```

Comment:
What's actually missing, now?

Reply:
We are missing functions declared in Rust like `graph_floyd_warshall`` and `digraph_floyd_warshall` because I planned to do it after. Just landing this PR was hard, adding 100+ function declarations would make it worse